### PR TITLE
test(threads): add trigger boundary eval fixtures

### LINKED ISSUE #115: test(threads): add trigger eval fixtures for boundary cases
## Problem

The `threads` skill has important positive and negative trigger boundaries: Codex-native subagents and GitHub queues should trigger it, but OS/language threads, email/forum threads, and OpenAI Assistants product threads should not.

## Proposed direction

- Add eval fixtures for explicit native-thread requests, GitHub queue requests, and review-only requests.
- Add negative fixtures for OS/language concurrency, chat/forum/email threads, and Assistants API thread questions.
- Include expected mode classification where practical: `single_agent`, `plan_only`, `execute_direct`, `review_only`, `research_spec`, or `clarify_first`.

## Acceptance

- Boundary fixtures are runnable with the repo's existing test/eval style.
- The fixtures do not require network or live GitHub state.
- Existing validation and unit tests continue to pass.

### PR DESCRIPTION
**Pull request by majiayu000**
## Summary
- Add local-only trigger eval fixtures for `threads` boundary cases in the `skill-creator/scripts/run_eval.py` schema: a top-level array with `query` and `should_trigger`.
- Cover positive Codex-native subagent, GitHub queue, review-only, research/spec, Chinese subagent, and plan-only prompts.
- Cover negative OS/language concurrency, discussion/email threads, Assistants API threads, and tiny single-agent tasks.
- Add a short `SKILL.md` pointer to the fixtures while keeping the router under the length-warning threshold.

Closes #115.

## Base
- `origin/main`: `5ba135ecee94732e9f078858b0bdad47f64d09f9`

## Verification
- `python3 -m json.tool skills/threads/evals/evals.json >/dev/null`
- runner-field check: all 12 cases have `query` and boolean `should_trigger`
- `wc -l skills/threads/evals/evals.json skills/threads/SKILL.md` -> `122` and `498`
- `python3 scripts/validate_skills.py --check`
- `python3 scripts/audit_skill_quality.py threads`
- `python3 -m unittest tests.test_threads_run_log -q`
- `python3 -m unittest discover -s tests -q`
- `git diff --check`

## Threads Evidence
- Queue gate thread: `019f02f8-b5b9-7ce2-b4cc-d71b0d6cf0e3`
- Coordinator selected #115 as the next independent issue tranche after resolving PR #120 review threads and closing stale #114.

### CODE REVIEW

File: skills/threads/evals/evals.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
+{
+  "skill_name": "threads",
+  "description": "Trigger-boundary fixtures for Codex-native subagent orchestration and GitHub issue/PR queue handling. These prompts are local-only and do not require live GitHub or network state.",
+  "evals": [
+    {
+      "id": 1,
+      "name": "explicit-codex-native-subagents",
+      "prompt": "Use threads to split this repository cleanup into independent Codex subagents with separate file ownership and review gates.",
```
[the remark this answers is not in the document]
Comment (majiayu000) [reply skills/threads/evals/evals.json]: Fixed in 43ae315 by converting `skills/threads/evals/evals.json` to the trigger-runner schema used by `skills/skill-creator/scripts/run_eval.py`: the file is now a top-level array and every item has `query` plus boolean `should_trigger`. I also verified the runner fields load for all 12 cases.
